src/main/java/ca/uhn/fhir/jpa/starter/BaseJpaRestfulServer.java

```diff
@@ -351,28 +354,24 @@ protected void initialize() throws ServletException {
     if (appProperties.getImplementationGuides() != null) {
       Map<String, AppProperties.ImplementationGuide> guides = appProperties.getImplementationGuides();
       for (Map.Entry<String, AppProperties.ImplementationGuide> guide : guides.entrySet()) {
-        packageInstallerSvc.install(new PackageInstallationSpec()
-          .setPackageUrl(guide.getValue().getUrl())
-          .setName(guide.getValue().getName())
-          .setVersion(guide.getValue().getVersion())
-          .setInstallMode(PackageInstallationSpec.InstallModeEnum.STORE_AND_INSTALL));
+			PackageInstallOutcomeJson outcome = packageInstallerSvc.install(new PackageInstallationSpec()
+				.setPackageUrl(guide.getValue().getUrl())
+				.setName(guide.getValue().getName())
+				.setVersion(guide.getValue().getVersion())
+				.setInstallMode(PackageInstallationSpec.InstallModeEnum.STORE_AND_INSTALL));
+
       }
     }
 
+    if(factory != null)
+    	interceptorService.registerInterceptor(factory.buildUsingStoredStructureDefinitions());
```

Comment:
Its just a style thing, but would appreciate braces around the if, even though its only one line.

Reply:
Fixed